hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: olm-artifacts-template
+
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  value: staging
+- name: IMAGE_TAG
+  value: latest
+
+objects:
+- apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: managed-upgrade-operator-catalog
+  spec:
+    sourceType: grpc
+    image: ${REGISTRY_IMG}:${CHANNEL}-${IMAGE_TAG}
+    displayName: managed-upgrade-operator Registry
+    publisher: SRE
```

Comment:
We should maybe be setting an affinity for infra nodes here, as suggested by OSD-4113. See cloud-ingress-operator as an example.

Reply:
Nice point Matt.. As per https://github.com/openshift/cloud-ingress-operator/commit/227c095f11de23183bcdc60d21345a296f0e19e0#diff-e12c68acf5db1ed8c169f6a9c8a722a6 I see that requiredDuringSchedulingIgnoredDuringExecution is used to schedule operator pod on infra node. Whereas, https://github.com/openshift/rbac-permissions-operator/pull/67/files#diff-5e5f804a8ee2fb8c81711b728eb5a21d is still a discussion where its suggested to use preferredDuringSchedulingIgnoredDuringExecution. I think its better to have a separate JIRA card for this and address it after this is decided. Sounds ok?

Reply:
Sure, a separate card sounds fine to me, let's get clarification on required vs preferred and then implement.